src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/SetAzureSqlServerAuditingPolicy.cs

```diff
@@ -42,12 +42,6 @@ public class SetAzureSqlServerAuditingPolicy : SqlDatabaseServerAuditingCmdletBa
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The set of the audit action groups")]
         public AuditActionGroups[] AuditActionGroup { get; set; }
 
-        /// <summary>
```

Comment:
It is not a breaking change. This is a removal of a mistakenly added parameter that is not supported on the server level.